docs/topics/add-rule-to-mta.adoc

```diff
@@ -9,8 +9,8 @@
 A {ProductName} rule is installed by copying the rule to the appropriate {ProductShortName} folder. {ProductShortName} scans for rules, which are files with the `.windup.xml` or `.mta.xml` extension in the following locations:
 
 * Directory specified by the `--userRulesDirectory` argument on the {ProductShortName} command line.
-* `<MTA_HOME>/rules/` directory. `<MTA_HOME>` is the directory where you install and run the {ProductName} executable.
-* `${user.home}/.mta/rules/` directory. This directory is created by {ProductShortName} the first time it is run. it contains rules, add-ons, and the {ProductShortName} log.
+* `<{ProductShortName}_HOME>/rules/` directory. `<{ProductShortName}_HOME>` is the directory where you install and run the {ProductName} executable.
+* `${user.home}/.{LC_PSN}/rules/` directory. This directory is created by {ProductShortName} the first time it is run. it contains rules, add-ons, and the {ProductShortName} log.
```

Comment:
someone verified that there is or will be a directory named $(user.home)/mtr/rules?

Reply:
@RichardHoch Someone did, although I am not sure how much attention they paid...